layouts/partials/fragments/nav.html

```diff
@@ -20,6 +23,23 @@
         {{- .Params.asset.text -}}
       </a>
     {{- end }}
+    {{- if .Params.search }}
+      <div class="search-container mr-3">
```

Comment:
Why not use "form-inline" from bootstrap? https://getbootstrap.com/docs/4.1/components/navbar/

Reply:
I still think it makes sense to use this. Might make upgrade to new bootstrap version less problematic and reduces styling we ourselves do.

Reply:
I think there is no styling for this input right now. We use `input-group`. It's documented in https://getbootstrap.com/docs/4.1/components/forms/#inline-forms

Reply:
Ok then happy to leave it as is.